### PR TITLE
Fixes #4228 by updating fr page vs en-US counterpart

### LINKED ISSUE #4228: Issue with "XSLT": Broken link on the documentation section
MDN URL: https://developer.mozilla.org/fr/docs/Web/XSLT

#### What information was incorrect, unhelpful, or incomplete?
The link "[Tutoriel XSLT et XPath](http://www.topxml.com/xsl/tutorials/intro/)" is broken and move the user on this game website (https://www.al3abmonkey.com/)

#### Specific section or headline?
The problem is in the documentation section

#### What did you expect to see?
I would like see a xslt tutorial website 

#### Did you test this? If so, how?
yes, just click on the link 

<details>
<summary>MDN Content page report details</summary>

* Folder: `fr/web/xslt`
* MDN URL: https://developer.mozilla.org/fr/docs/Web/XSLT
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/fr/web/xslt/index.md
* Last commit: https://github.com/mdn/translated-content/commit/4453c14499fb94b0da642ccfd3a48baf10cdf642
* Document last modified: 2022-02-18T06:47:52.000Z

</details>

### PR DESCRIPTION
**Pull request by SphinxKnight**
Fixes #4228

### CODE REVIEW

File: files/fr/web/xslt/index.md [hunk 1]
```diff
@@ -1,149 +1,57 @@
 ---
 title: XSLT
 slug: Web/XSLT
-tags:
-  - XSLT
+
```
Comment (cw118) [inline files/fr/web/xslt/index.md:4]:
```suggestion
```

File: files/fr/web/xslt/index.md [hunk 2]
```diff
@@ -1,149 +1,57 @@
...
+XSLT est le plus souvent utilisé pour convertir des données entre différents schémas XML ou pour convertir des données XML en pages web ou en documents PDF.
 
-L'objectif principal est la transformation d'un document [XML](/fr/XML "fr/XML") vers un autre, ou un dialecte [XML](/fr/XML "fr/XML") (XHTML, XSL-FO, [HTML](/fr/HTML "fr/HTML"), etc.). Cependant, le langage XSLT permet aussi les transformations vers tout autre type de document, au format texte ou dans un format binaire (bien que ceci ne soit pas nativement prévu par la recommandation XSLT).
+## Documentation
 
-XSLT s'appuie sur XPath (une autre partie de la recommandation XSL) pour désigner une partie d'un arbre [XML](/fr/XML "fr/XML").
+- [Référence des éléments XSLT](/fr/docs/Web/XSLT/Element)
+  - : La référence indiquant la documentation pour chaque élément XSLT.
+- [Transformer du XML avec XSLT](/fr/docs/Web/XSLT/Transforming_XML_with_XSLT)
+  - : XSLT permet d'utiliser une feuille de style pour transformer un document XML intial de deux façons&nbsp;: manipuler et trier le contenu (y compris en réordonnant l'ensemble du document si besoin) d'une part et transformer le contenu dans un format différent d'autre part.
```
Comment (cw118) [inline files/fr/web/xslt/index.md:42]:
```suggestion
  - : XSLT permet d'utiliser une feuille de style pour transformer un document XML initial de deux façons&nbsp;: manipuler et trier le contenu (y compris en réordonnant l'ensemble du document si besoin) d'une part et transformer le contenu dans un format différent d'autre part.
```

**Review by cw118 — CHANGES_REQUESTED** [review]
Deux petites coquilles :)

**Comment by SphinxKnight** [comment]
Merci @cw118 🙇‍♂️

**Review by cw118 — APPROVED** [review]
👍

**Pull request merged**